### DISCUSSION TITLE
Unify the Automation and Scripts 3-dot Menu

### DISCUSSION THREAD

**Discussion by oneseventhree** [home-assistant/frontend#15314] [Automations / scripts / scenes] [answered]
Hi,

It would be great if the automations and script 3-dot menus looked and functioned the same:

Automations:
<img width="215" alt="Screenshot 2023-02-02 at 6 12 12 pm" src="https://user-images.githubusercontent.com/15043282/216259555-ffcb46cf-1e36-43c7-96ef-73ed3ad76f61.png">

Script:
<img width="212" alt="Screenshot 2023-02-02 at 6 11 54 pm" src="https://user-images.githubusercontent.com/15043282/216259576-9cb563cb-e0fc-4c6e-a609-49c48156a991.png">

I really like how you can add a description to Automations and wish you could for Scripts as well:
<img width="436" alt="Screenshot 2023-02-02 at 6 10 54 pm" src="https://user-images.githubusercontent.com/15043282/216259600-f3e484c2-241f-41cc-8815-9cacb667051d.png">

As you can see I use descriptions as a quick reference guide on what the different Choose Options in my script does.

Thank you in advance :)

**Comment by stiltjack** [comment]
+1 for this.

You can add descriptions to scripts at the moment, but only in yaml - they are not displayed in the UI.

In addition, you can use line feeds and markdown in automation descriptions; not in script descriptions, where text is displayed (in yaml) in a single block.

**Comment by karwosts** [comment, marked as answer]
Completed via #20791

**Reply by stiltjack** [reply]
Thanks for the update, but descriptions in scripts still do not support line feeds or markdown as they do in automations - indeed, using punctuation can produce odd results as it's interpreted as yaml - try ending a description with "script."

**Reply by karwosts** [reply]
I don't agree, that is all supported? If you've recently updated make sure to clear your browser cache.

**Reply by stiltjack** [reply]
That was it - thank you!
